app/src/main/java/com/example/findu/presentation/ui/home/component/HomeReportCard.kt

```diff
@@ -58,30 +63,31 @@ fun HomeReportCard(
         )
         Spacer(modifier = Modifier.height(14.dp))
         HomeReportCardIndicator(
-            modifier = Modifier.padding(horizontal = 15.dp), indicatorClicked = indicatorClicked,
+            modifier = Modifier.padding(horizontal = 15.dp),
+            indicatorClicked = indicatorClicked,
             selected = homeReportDuration
```

Comment:
변수명이 모호한 거 같습니다!
두 변수명이 클릭 이벤트인지, Boolean 값인지, 한 눈에 파악되지 않는 것 같습니다!
```suggestion
            onIndicatorSelected = onIndicatorSelected,
            selectedDuration = homeReportDuration